### PR TITLE
Support workload pack groups for repair, download, and show URL operations

### PR DESCRIPTION
**Pull request by dsplaisted**

### CODE REVIEW

File: src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs [hunk 1]
```diff
@@ -476,6 +482,91 @@ public void Shutdown()
...
+                    {
+                        FileName = "msiexec",
+                        ArgumentList =
+                        {
+                            "/a",
+                            msiPath,
+                            $"TARGETDIR={msiExtractionPath}",
+                            "/QN",  //  Quiet: No UI
+                        }
+                    };
```
Comment (joeloff) [inline src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs:536]: You should be able to get rid of this and use the native APIs we have in the CLI. `WindowsInstaller.InstallProduct` allows you to do that. The doc comments actually mentions admin installs.

**Comment by dsplaisted** [comment]
@joeloff @gkulin I still have to add localizable strings for some error messages, but otherwise I think this is ready for review.

File: src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs [hunk 2]
```diff
@@ -522,21 +521,12 @@ public async Task ExtractManifestAsync(string nupkgPath, string targetPath)
...
-                            msiPath,
-                            $"TARGETDIR={msiExtractionPath}",
-                            "/QN",  //  Quiet: No UI
-                        }
-                    };
 
-                    var exitCode = psi.Execute();
-                    if (exitCode != 0)
+
+                    _ = WindowsInstaller.SetInternalUI(InstallUILevel.None);
```
Comment (joeloff) [inline src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs:526]: Did this work before without creating the directory first when you used MSIEXEC? Admin installs usually require having the TARGETDIR created before doing the install.

**Review by joeloff — APPROVED** [review]

File: src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```
Comment (gkulin) [inline src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs:3]: nit: I'm guessing you didn't mean to add an extra commented line here

**Review by gkulin — APPROVED** [review]

File: src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs (hunk 3 above)
Comment (joeloff) [reply src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs:3]: 👍 That's been bothering me for a while now, especially when creating new files in VS. I think it comes from https://github.com/dotnet/sdk/blob/60feccb2da91f75c7ec6a58ee2f600b27f4a1f85/.editorconfig#L159

**Review by joeloff — APPROVED** [review]
Loc change look good

**Pull request merged**